subsys/esb/Kconfig

```
@@ -141,6 +146,18 @@ config ESB_DYNAMIC_INTERRUPTS
 	  interrupts. This allows reconfiguring ESB_SYS_TIMER_IRQn, ESB_EVT_IRQ,
 	  and RADIO_IRQn handlers during runtime when ESB is uninitialized.
 
+config ESB_MPSL_TIMESLOT
+	bool "Use ESB on MPSL Timeslot"
```

minor nit.
Avoid `Use` in the prompt.
```suggestion
	bool "ESB on MPSL Timeslot"
```